tests/accounts/nuls2.test.ts

```diff
@@ -70,9 +70,26 @@ describe("NULS2 accounts", () => {
         const account = await nuls2.ImportAccountFromPrivateKey(testPrivateKey);
         const msg = Buffer.from("Nuuullss2");
 
-        const c = account.encrypt(msg);
+        const c = await account.encrypt(msg);
         const d = account.decrypt(c);
         expect(c).not.toBe(msg);
         expect(d).toStrictEqual(msg);
     });
+
+    it("Should delegate encrypt and decrypt content with NULS2", async () => {
+        const accountA = await nuls2.ImportAccountFromPrivateKey(testPrivateKey);
+        const accountB = await nuls2.ImportAccountFromPrivateKey(testPrivateKey);
```

Comment:
They're the same account?

Reply:
Might be better with this: https://github.com/aleph-im/aleph-sdk-ts/pull/104/commits/b8dffb1a10540b1c169062fba1f725e0c0ed5e6c